modules/home/transformations/filter-plugins.php

```diff
@@ -30,17 +29,59 @@ private function get_installed_plugins(): array {
 		return array_keys( $plugins );
 	}
 
+	private function is_plugin_activated( $plugin_path ) {
+		return $this->wordpress_adapter->is_plugin_active( $plugin_path );
+	}
+
+	private function get_install_plugin_url( $plugin_path ) {
```

Comment:
@louiswol94 Could or should we extend these functions from the admin apps page?

Reply:
@hein-obox Yes please extract this somehow so that it is shared between the two modules.

Reply:
I have extracted it to an `Elementor_Adapter` class.
My main questions about my approach: 

- Is it okay that I have created a general Elementor adapter for this?
- The adapter methods are public. Is that okay?